spec/factories/claims.rb

```diff
@@ -11,7 +11,8 @@
 
     after(:build) do |claim, evaluator|
       claim.eligibility = build(*evaluator.eligibility_factory) unless claim.eligibility
-      claim.academic_year = "2019/2020" unless claim.academic_year_before_type_cast
+      claim_academic_year = claim.policy == EarlyCareerPayments ? AcademicYear.new([*Date.today.year..Date.today.year + 4].sample) : "2019/2020"
```

Comment:
Actually thought this should be converted to string for factories. Need to test this cause I've used strings elsewhere but the actual object would be better.

Reply:
Did a quit test and these should be serialized values at DB level, ie, strings.